Sources/Orbit/Components/Alert.swift

```diff
@@ -12,284 +12,195 @@ import SwiftUI
+            EmptyView()
+        } content: {
+            EmptyView()
+        } icon: {
+            Icon(icon)
         }
     }
 
-    @ViewBuilder var iconContent: some View {
-        icon
-            .font(.system(size: Orbit.Icon.Size.normal.value))
-            .foregroundColor(iconColor ?? outlineColor)
-            .iconColor(iconColor ?? outlineColor)
-            .accessibility(.alertIcon)
-    }
-
-    @ViewBuilder var titleContent: some View {
-        Text(title)
-            .bold()
-            .accessibility(.alertTitle)
-    }
-
-    @ViewBuilder var defaultButtons: some View {
-        switch buttons {
-            case .primary, .secondary, .primaryAndSecondary:
-                // Keeping the identity of buttons for correct animations
-                HStack(alignment: .top, spacing: .small) {
-                    switch buttons {
-                        case .primary(let primaryButton),
-                             .primaryAndSecondary(let primaryButton, _):
-                            Button(primaryButton.label, type: primaryButtonType, action: primaryButton.action)
-                                .accessibility(.alertButtonPrimary)
-                        case .none, .secondary, .inline:
-                            EmptyView()
-                    }
-
-                    switch buttons {
-                        case .secondary(let secondaryButton),
-                             .primaryAndSecondary(_, let secondaryButton):
-                            Button(secondaryButton.label, type: secondaryButtonType, action: secondaryButton.action)
-                                .accessibility(.alertButtonSecondary)
-                        case .none, .primary, .inline:
-                            EmptyView()
-                    }
-                }
-                .buttonSize(.compact)
-            case .none, .inline:
-                EmptyView()
+    /// Creates Orbit Alert component with custom content and icon.
+    init(
+        _ title: String = "",
+        description: String = "",
+        isSubtle: Bool = false,
+        @AlertButtonsBuilder buttons: () -> Buttons,
+        @ViewBuilder content: () -> Content = { EmptyView() },
+        @ViewBuilder icon: () -> Icon = { EmptyView() }
+    ) {
+        self.init(title: title, description: description, isSubtle: isSubtle) {
+            buttons()
+        } content: {
+            content()
+        } icon: {
+            icon()
         }
     }
+}
 
-    @ViewBuilder var background: some View {
-        backgroundColor
-            .overlay(
-                RoundedRectangle(cornerRadius: BorderRadius.default)
-                    .strokeBorder(strokeColor, lineWidth: 1)
-            )
-            .overlay(
-                outlineColor
-                    .frame(height: 3),
-                alignment: .top
-            )
-    }
+// MARK: - Types
+struct AlertButtonStyle: PrimitiveButtonStyle {
```

Comment:
This variant can be made public so the call site can replicate more custom layout using these alert buttons. The inline variant may stay internal or can be made public too.

Reply:
I was thinking of scenario where an Alert should show some custom content "below" buttons and a call site would need to be able to create these buttons for such specific layout. It should not do any harm in exposing it. If not, we can remove the public from one of the `makeBody` methods.

Reply:
Done.